python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1058,6 +1060,12 @@ def _combineExposureMetadata(self, visit, dataRefs, isGen3=True):
             rec['raCorners'][:] = [sph.getRa().asDegrees() for sph in sph_pts]
             rec['decCorners'][:] = [sph.getDec().asDegrees() for sph in sph_pts]
 
+        metadata = dafBase.PropertyList()
+        metadata.add("COMMENT", "Catalog id is detector id, sorted.")
+        metadata.add("COMMENT", "Only detectors with data have entries.")
```

Comment:
I guess this latter comment is true in this case because we're iterating over dataRefs. It wasn't entirely obvious to me at first.